metadata.tvdb.com.python/resources/lib/test_imdb_rating.py

```diff
@@ -0,0 +1,8 @@
+import pytest
```

Comment:
@Razzeee this needs to be added to .gitattributes

Reply:
Should be better now